apps/editors/helpers.py

```diff
@@ -116,9 +121,18 @@ def editors_breadcrumbs(context, queue=None, addon_queue=None, items=None,
             'flagged_themes': _('Flagged Themes'),
             'rereview_themes': _('Update Themes'),
         }
+        if unlisted:
+            queues = {
```

Comment:
I think I'd prefer an if-else here. It's not immediately obvious what's happening on first read.